python/pyarrow/_parquet.pyx

```diff
@@ -187,86 +175,72 @@ cdef class ColumnChunkMetaData:
                                           self.total_compressed_size,
                                           self.total_uncompressed_size)
 
-    property file_offset:
-
-        def __get__(self):
-            return self.metadata.file_offset()
-
-    property file_path:
-
-        def __get__(self):
-            return frombytes(self.metadata.file_path())
-
-    property type:
-
-        def __get__(self):
-            return physical_type_name_from_enum(self.metadata.type())
-
-    property num_values:
-
-        def __get__(self):
-            return self.metadata.num_values()
-
-    property path_in_schema:
-
-        def __get__(self):
-            path = self.metadata.path_in_schema().get().ToDotString()
-            return frombytes(path)
-
-    property is_stats_set:
-
-        def __get__(self):
-            return self.metadata.is_stats_set()
-
-    property statistics:
-
-        def __get__(self):
-            if not self.metadata.is_stats_set():
-                return None
-            statistics = RowGroupStatistics()
-            statistics.init(self.metadata.statistics())
-            return statistics
-
-    property compression:
-
-        def __get__(self):
-            return self.metadata.compression()
+    @property
+    def file_offset(self):
+        return self.metadata.file_offset()
 
-    property encodings:
+    @property
+    def file_path(self):
+        return frombytes(self.metadata.file_path())
 
-        def __get__(self):
-            return map(encoding_name_from_enum,
-                       self.metadata.encodings())
+    @property
+    def type(self):
```

Comment:
Shouldn't this be called `pshysical_type` like in `RowGroupStatistics`?

Reply:
Yes that would be much better